### ISSUE TITLE
Stops: Map is not working 

### ISSUE THREAD

**Issue by antoine29** [WRI-Cities/static-GTFS-manager#148]
#### Operating system

Debian 10 x64
#### Python version 

Running from docker container.

#### Problem description

I was using StaticGTFSManagar normally, but now I can't see the map in stops page, I'm able to see or add more stops but the map is in blank. 
#### Console / terminal output if any

Since the browser logs the next error:

> https://api.mapbox.com/styles/v1/nikhilsheth/cj8rdd7wu45nl2sps9teusbbr/tiles/256/13/4262/3599?access_token=pk.eyJ1IjoibmlraGlsc2hldGgiLCJhIjoiQTREVlJuOCJ9.YpMpVVbkxOFZW-bEq1_LIw 401 (Unauthorized)
> 

I think is something related with the mapbox keys, maybe could be a good idea let users set its own mapbox api keys.
#### Expected Action

Show a map in stops page.
#### Details of data you are working with, if relevant
Q: Is there anything peculiar about the data you are working with, which may be causing the issue? Any extra files in the feed? Extra columns? Any mandatory columns/fields omitted?  
A: Nothing special, is a new dataset built using staticGtfsManager.
Q: What is the size of the dataset : how many lines in stops.txt, routes.txt and stop_times.txt?  
A: It's very tiny data set, less than 100 stops.
Q: Are you starting a new feed from scratch and haven't filled in any of the other files like calendar.txt?  
A: no
#### Anything else

**Comment by answerquest** [comment]
Hi @antoine29 thanks a bunch for flagging this. Yes, the mapbox key ran out of juice now that their usage terms have changed. I'd found free alternatives but hadn't changed the stops page yet. Updating the repo in just a bit.

**Comment by answerquest** [comment]
Hi @antoine29 can you download the latest code and try again? I've changed the map layers in all the pages now - no more API keys etc, and I found great free options.

**Comment by antoine29** [comment]
Hi @answerquest 
I pulled the new changes one week ago and the maps was working properly. Also I figured out the csv databanks feature and the new map layers in the stops page, this last feature is awesome!
This tool is helping me with a college project so I just want to say thank you and Great work!
![Screenshot from 2019-08-06 13-28-22](https://user-images.githubusercontent.com/28495651/62562053-69976480-b84e-11e9-821e-33127e7cf9d3.png)
